src/phoenix/server/api/types/Project.py

```diff
@@ -33,6 +33,10 @@ def end_time(self) -> Optional[datetime]:
     def record_count(self) -> int:
         return self.project.span_count
 
+    @strawberry.field
+    def trace_count(self) -> int:
+        return self.project.trace_count
```

Comment:
As a follow-up I think I'm gonna need to use a time range here so I can just show the trace count for the last 7 days.